hasura/metadata/databases/bdjuno/tables/public_transaction.yaml

```
@@ -8,11 +8,14 @@ object_relationships:
 array_relationships:
 - name: messagesByTransactionHash
   using:
-    foreign_key_constraint_on:
-      column: transaction_hash
-      table:
-        name: message
+    manual_configuration:
+      remote_table:
         schema: public
+        name: message
+      insertion_order:
+      column_mapping:
+        transaction_hash: hash
+        partition_id: partition_id
```

```suggestion
      column_mapping:
        transaction_hash: hash
```
partition_id in column mapping should be dropped for the messageByAddress function to work properly